tests/v2/validation/provisioning/rke2/agent_customization_test.go

```diff
@@ -0,0 +1,244 @@
+//go:build validation
```

Comment:
Let's set up a meeting on recurring run expectations as well. There has been new provisioning tests added that don't run in our recurring runs at all, because they're either dynamic or the right tag isn't added.